user-guide/reducing-processing-time.dita

```diff
@@ -35,7 +35,8 @@
       </section>
       <section>
          <title>Use the latest Java version</title>
-         <p>DITA-OT 2.0 requires Java 7, but using the latest version Java 8 will further reduce processing time.</p>
+         <p>DITA-OT 2.0 to 2.3 require Java Runtime Environment (JRE) 7, but using JRE 8 with these versions can
```

Comment:
Java 8 is not the latest Java-SE release anymore. Thus this should still say "latest version".